docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,9 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+- **DAML Standard Library**: Move the ``Tuple`` and ``Either`` types to ``daml-prim:DA.Types`` rather
+  than exposing internal locations
```

Comment:
We should also leave an explanation how users need to update their applications on the Java/Scala/... side. cc @bethaitman

Reply:
We also need to update https://github.com/digital-asset/daml/blob/master/docs/source/app-dev/daml-lf-translation.rst#tuple-types